apps/silverback-drupal/composer.json

```diff
@@ -102,5 +102,9 @@
             "if [ -d web/modules/custom ]; then phpunit web/modules/custom; fi",
             "yarn install && CYPRESS_TAGS=@COMPLETED cypress run"
         ]
+    },
+    "require-dev": {
+        "mglaman/phpstan-drupal": "^0.12.5",
+        "phpstan/extension-installer": "^1.0"
```

Comment:
This can be removed once `drupal/cypress` is released and the above dependencies are visible for composer from the package metadata.
(Composer loads the package from a local directory, but it reads the list of dependencies from the Packagist and there is no way to force it to read metadata from the local package `composer.json`)